akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -9,6 +9,7 @@ import scala.util.control.NonFatal
 import akka.stream.stage._
 import akka.stream.Supervision
 import akka.stream.impl.ReactiveStreamsCompliance
+import akka.stream.FlowMaterializer
```

Comment:
That is not the same, because now interpreter relies on the FlowMaterializer interface. I like a more layered approach, even if this is not a separate subproject.